requirements-docs.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx==1.1.3
+-rrequirements.txt
```

Comment:
I was just explaining why we do it for boto3. It is because read the docs shows the develop branch of boto3 and so sometimes to build the docs off of the develop branch we may need the develop branch of botocore.

For the CLI we do not use the develop branch (latest release instead). So yes it makes more sense to do ``pip install -e .`` I will make the change